rudder-webapp/SOURCES/rudder-metrics-reporting

```diff
@@ -55,7 +58,7 @@ def metrics():
   data = { }
 
   # Define the versionning of the file
-  data['file_version'] = "2"
+  data['file_version'] = "3"
```

Comment:
we should maybe do this in 6.2

Reply:
No, I think it's ok, because the user of that file/API is us - so the burden is on our side.